tools/link.py

```diff
@@ -2098,6 +2098,9 @@ def phase_final_emitting(options, state, target, wasm_target):
 
   target_dir = os.path.dirname(os.path.abspath(target))
 
+  if settings.PTHREADS:
+    replace_worker_options_placeholders(final_js)
```

Comment:
Based on test failures, is there an early location where we should be replacing these values? 🤔

Reply:
After further reflection on how these files fit together, it seems some kind of internal setting could be a more appropriate way to configure this? (i.e. [settings_internal.js](https://github.com/emscripten-core/emscripten/blob/main/src/settings_internal.js))

Reply:
Solved thanks to the maintainers suggested improvement.